README.md

```diff
@@ -1,2 +1,27 @@
 # BridgeServer2-infra
-CloudFormation templates for BridgeServer2
+Install and configure the AWS Infrastructure for the BridgeServer 2.0 application.
+
+
+## Initialize a new stack
+1. Deploy the infrastructure to AWS. Run `sceptre launch-stack develop bridgeserver2`
+
+The above should create the infrastructure required for BridgeServer2, it does not actually deploy the app it only installs and configures the required AWS infrastructure for it. The AWS appilcation health check is enabled therefore the EB environment will start in an error state since the app has not been deployed. The environment status should turn green once the app is successfully deployed.
+
+2. Check for additional bootstrapping steps in [BridgeServer2 repo](https://github.com/Sage-Bionetworks/BridgeServer2).
```

Comment:
No, BridgeServer2 is intended to fully replace BridgePF. It's a separate stack and currently has its own bootstrapping and setup (which is currently empty, since it's a minimal passthrough service at the moment).